src/ecrc-frontend/src/components/page/termsOfUsePage/TermsOfUsePage.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
```

Comment:
lets call this folder `termsOfUse` and the file `TermsOfUse` instead of `termsOfUsePage` 
the fact that it resides inside the `page` directory identifies it as a page component :)

Reply:
the reason is there is base component called TermsOfUse

Reply:
want to differentiate them

Reply:
thoughts?

Reply:
that is what I am saying - the fact that it resides in a folder called `page` already takes care of that. the other one is a base component so it resides inside directory called `base`, this one should be named the same but inside `page`

Reply:
OK. But I can change that.

Reply:
ok. will change.